### ISSUE TITLE
How to handle differences in translations between mobile and desktop

### ISSUE THREAD

**Issue by hellais** [ooni/probe#970]
We discussed with @lorenzoPrimi and @sarathms during the weekly frontend meeting that we should eventually consider how to handle difference in translations between mobile and desktop.

Currently we have the same set of strings which are shared across both mobile and desktop, but there may be certain cases in which we want different strings based on the context.

For example on mobile you may say "please **tap** on the button", while on desktop you would say "please **click** on the button".

It's worth noting that if we have separate sets of resources, transifex should in theory handle this properly in virtue of it's translation memory: https://docs.transifex.com/translation-memory/tm-introduction.

We said that as an experiment we should at some point look into uploading a new resource and seeing how well this performs.

**Comment by hellais** [comment]
We should also consider even splitting Android and iOS strings into two.

Apparently these also change depending on the platform as reported on transifex:

> Indeed, iOS/macOs refers to settings in French as "Réglages", "Paramètres" for Android/Google.
> Here are two examples from the horse’s mouth:
> https://support.apple.com/fr-be/HT204204
> https://support.google.com/accounts/answer/3118621?hl=fr

**Comment by lorenzoPrimi** [comment]
I don't think we should split the string file, but I think we should tag every string with android,ios,desktop... etc and then edit the download file to download only the relative tagged string

**Comment by agrabeli** [comment]
@lorenzoPrimi while that's a great suggestion, I wonder how obvious it will be for translators to do that (to translate files based on tags)? Perhaps an "easier" option could be to make a copy of the current file, rename the copy to "desktop", and just edit the different strings directly in there (I could do this editing)? That way it would be clearer to translators that those strings are for desktop. However, I wonder if there's a way of keeping the mutual strings (across "mobile" and "desktop" files) in sync (so that translators don't need to re-translate the same strings on desktop)?

**Comment by lorenzoPrimi** [comment]
I've done my part.
Labels used:
- FALSE (String not used in the app)
- DEPRECATED (String used in the app but can be removed soon)
- NEW (String used in an alpha version of the app)
